browser/renderer_context_menu/test/render_view_context_menu_browsertest.cc

```diff
@@ -0,0 +1,93 @@
+#include "brave/app/brave_command_ids.h"
```

Comment:
need copyright. You can see the right form by running "npm run presubmit"

Reply:
Done.